src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1115,6 +1133,16 @@ internal static partial class Library
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: Sqrt)
             },
+            {
+                BuiltinFunctionsCore.SqrtT,
```

Comment:
>BuiltinFunctionsCore.SqrtT, [](http://example.com/codeflow?start=16&length=27)

To double check - are we adding this overload to intrepreter?

Reply:
Yes